corehq/apps/data_interfaces/static/data_interfaces/js/case_dedupe_main.js

```diff
@@ -82,6 +83,11 @@ hqDefine("data_interfaces/js/case_dedupe_main", [
     };
 
     $(function () {
+        // This is a little hacky; it prevents the "multiple bindings to same
+        // element" error.
```

Comment:
...looking into some solution like [this](https://stackoverflow.com/a/31046743) at the moment...